warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1248,6 +1249,8 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
                 ParallelIndexExpansion regexExpansion = new ParallelIndexExpansion(config, scannerFactory, metadataHelper, expansionFields,
                                 config.isExpandFields(), config.isExpandValues(), config.isExpandUnfieldedNegations());
                 queryTree = (ASTJexlScript) regexExpansion.visit(queryTree, null);
+                // regex expansion picks up an extra set of parens, so quickly fix that here
```

Comment:
Can we fix the regex expansion to avoid the extra parens in the first place?

Reply:
An early iteration attempted to fix that but it's uhm, ugly.